tests/php/Model/Delete/DeleteRecordPolicyTest.php

```diff
@@ -0,0 +1,176 @@
+<?php
+
+namespace TractorCow\Fluent\Tests\Model\Delete;
+
+use SilverStripe\Core\Injector\Injector;
+use SilverStripe\Dev\SapphireTest;
+use SilverStripe\ORM\DB;
+use SilverStripe\ORM\ValidationException;
+use TractorCow\Fluent\Model\Delete\DeletePolicy;
+use TractorCow\Fluent\Model\Delete\DeleteRecordPolicy;
+use TractorCow\Fluent\Model\Domain;
+use TractorCow\Fluent\Model\Locale;
+use TractorCow\Fluent\State\FluentState;
+use TractorCow\Fluent\Tests\Model\Delete\Fixtures\FilteredRecord;
+use TractorCow\Fluent\Tests\Model\Delete\Fixtures\LocalisedRecord;
+use TractorCow\Fluent\Tests\Model\Delete\Fixtures\Record;
+
+class DeleteRecordPolicyTest extends SapphireTest
+{
+    protected static $fixture_file = '../LocaleTest.yml';
+
+    protected static $extra_dataobjects = [
+        Record::class,
+        LocalisedRecord::class,
+        FilteredRecord::class,
+    ];
+
+    protected function setUp()
+    {
+        parent::setUp();
+
+        // Clear cache
+        Locale::clearCached();
+        Domain::clearCached();
+
+        // Set default
+        FluentState::singleton()
+            ->setLocale('es_US')
+            ->setIsDomainMode(false);
+    }
+
+    /**
+     * @throws ValidationException
```

Comment:
What's this for? Is this a PHPStorm addition, or an expected exception?

Reply:
First lol.